meta/main.yml

```diff
@@ -1,11 +1,14 @@
 ---
 dependencies: []
 galaxy_info:
-  author: First Last
+  author: Alexander King
   company: CISA Cyber Assessments
-  description: Skeleton Ansible role
+  description: This role installs the Admiral Python library for Certificate Transparency log scanning.
```

Comment:
This should be reworded because all this role is doing is installing the Docker composition for running the Admiral.

Reply:
It looks like you rebased and that commit is _gone_ but in https://github.com/cisagov/ansible-role-admiral/pull/3/commits/2abf06f4a1268e32ee58601768389808031362ff it appears to be correctly doing what I asked.